x-pack/plugins/triggers_actions_ui/public/application/sections/alert_summary_widget/components/alert_summary_widget_full_size.stories.tsx

```diff
@@ -23,3 +23,14 @@ export const FullSize = {
     },
   },
 };
+
+export const FullSizeWithoutChart = {
```

Comment:
nit: I don't think we need a new story for this, we can add the hide chart prop to the `FullSize` story.

Reply:
Yes, true, removed 👍